hack/debug-network.sh

```diff
@@ -0,0 +1,729 @@
+#!/bin/bash
+
+# This script will find any network connectivity issues with OpenShift SDN.
+# It performs: pod<->pod, pod<->node, pod<->service, pod<->external checks.
+# Assumes: SSH access to openshift master and openshift nodes.
+
+# die() echoes args to stderr and exits
+function die() {
+    echo "$@" 1>&2
+    exit 1
+}
+readonly -f die
+
+# echo_and_eval() echoes the command provided as $@ and then runs it
+function echo_and_eval() {
```

Comment:
I understand this is a migration of an existing script, but I still don't feel convinced of the need for this. `eval` has a lot of rough edges, so the bar for doing this over `set -x` is pretty high in my opinion.